File: app/src/main/java/com/kamontat/checkidnumber/adapter/ViewPagerAdapter.java
package com.kamontat.checkidnumber.adapter;

import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentPagerAdapter;

import java.util.*;

/**
 * @author kamontat
 * @version 1.0
 * @since Thu 11/May/2017 - 10:19 PM
 */
public class ViewPagerAdapter extends FragmentPagerAdapter {
	private final List<Fragment> fragmentList;
	
	public ViewPagerAdapter(FragmentManager fm) {
		super(fm);
		this.fragmentList = new ArrayList<>();
	}
	
	public void addFragment(Fragment fragment) {
		fragmentList.add(fragment);
	}
	
	@Override
	public Fragment getItem(int position) {
		return fragmentList.get(position);
	}
	
	@Override
	public int getCount() {
		return fragmentList.size();
	}
}

File: app/src/main/java/com/kamontat/checkidnumber/adapter/ViewPagerAdapter.kt
package com.kamontat.checkidnumber.adapter

import android.support.v4.app.Fragment
import android.support.v4.app.FragmentManager
import android.support.v4.app.FragmentStatePagerAdapter

import java.util.*

/**
 * @author kamontat
 * *
 * @version 1.0
 * *
 * @since Thu 11/May/2017 - 10:19 PM
 */
class ViewPagerAdapter(fm: FragmentManager) : FragmentStatePagerAdapter(fm) {
    private val fragmentList: MutableList<Fragment>

    init {
        this.fragmentList = ArrayList<Fragment>()
    }

    fun addFragment(fragment: Fragment) {
        fragmentList.add(fragment)
    }

    override fun getItem(position: Int): Fragment {
        return fragmentList[position]
    }

    override fun getCount(): Int {
        return fragmentList.size
    }
}
